_data/meltano/utilities/metabase/metabase.yml

```diff
@@ -0,0 +1,40 @@
+name: metabase
+namespace: metabase
+label: Metabase
+description: Metabase is the easy, open-source way for everyone in your company to ask questions and learn from data.
+docs: https://www.metabase.com/
+repo: https://github.com/metabase/metabase
+maintenance_status: active
+variant: metabase
+pip_url: ''
```

Comment:
```suggestion
```

Distinction of leaving out entirely vs leaving blank.

@pnadolny13 - Can you try this way with the field left out?